bindings/rust/bench/src/bin/memory.rs

```diff
@@ -108,7 +108,7 @@ fn memory_bench<T: TlsConnection>(opt: &Opt) -> Result<(), Box<dyn Error>> {
             )?;
             conn_pair = TlsConnPair::wrap(client_conn, server_conn);
         } else {
-            conn_pair = TlsConnPair::<T, T>::new(
+            conn_pair = TlsConnPair::<T, T>::from_configs(
                 CryptoConfig::default(),
                 HandshakeType::default(),
                 buffers.pop().unwrap(),
```

Comment:
How does this work? Doesn't `from_configs` accept a client and server config?

Reply:
Hmm, well, I matched the previous behavior but it looks like the previous behavior was broken. 😦 

We should probably just remove the memory benchmarking functionality, or at least add it do CI